### ISSUE TITLE
ImagePicka problem with Twitter pictures

### ISSUE THREAD

**Issue by aprunai** [eight04/image-picka#257]
After the recent update, attempting to save pictures from Twitter - both unzoomed and zoomed - returns the "NetworkError when attempting to fetch resource" error message if "try to download images from browser cache" is enabled. Sadly I need that setting for other sites like Pixiv.
I didn't notice this behavior in any other site for now.

**Comment by eight04** [comment]
Tested on https://twitter.com/davidwalshblog/status/1364660145364606984 and I can't reproduce with Firefox 87.0b2 + Image Picka 0.14.0.

What is your browser? Does it work in a new profile?

**Comment by aprunai** [comment]
Sorry for not specifying. I'm on Firefox 85.0.1 64-bit on Linux Mint. ImagePicka is version 0.14.0.

**Comment by aprunai** [comment]
So, I tried with all my extensions disabled, and the behavior continued. Disabling Firefox's anti-tracking protection didn't help either.
Then I created a new profile, and everything worked fine. So it must be my profile's fault.
I'm gonna close the issue since it's not really related to ImagePicka.
BTW: thank you very much for developing this extension! It's really helpful, I couldn't browse comfortably without it!!
